### PR TITLE
fix(styles): fix typography bugs

### PR DESCRIPTION
**Pull request by followyouhome**
### What was a problem?

After removal of entire stylesheet some fonts disappeard

### CODE REVIEW

**Comment by ZengJunyong** [comment]
Failed to build

**Comment by followyouhome** [comment]
Updated, please review

File: packages/recomponents/src/styles/typography.scss [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    src: url('/static/fonts/RobotoMono-Regular.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/RobotoMono-Regular.woff2') format('woff2'),
+        url('/static/fonts/RobotoMono-Regular.woff') format('woff'),
+        url('/static/fonts/RobotoMono-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+h1, .h1 {
+    font-size: $h1;
```
Comment (mathewsonke) [inline packages/recomponents/src/styles/typography.scss]: In the future, we'll want to allow customizing `font-size` and `font-weight` with CSS vars.

**Pull request merged**